options/locale/locale_en-US.ini

```diff
@@ -1463,6 +1463,9 @@ issues.attachment.open_tab = `Click to see "%s" in a new tab`
 issues.attachment.download = `Click to download "%s"`
 issues.subscribe = Subscribe
 issues.unsubscribe = Unsubscribe
+issues.pin = Pin
+issues.unpin = Unpin
```

Comment:
Move these to to top-level please, outside any section, e.g. translation keys `pin` and `unpin`.

Reply:
You mean `pin` instead of `repo.issues.pin`? Why? They belong to the Issues of a Repo.

Reply:
Yes, but these terms are universal and usable elsewhere, like "pin" for project card, etc. We need to cut down on translating the same thing 5 times, unless they mean different things in different contexts, of course.